docs/content/using/changing_host_connectivity.md

```diff
@@ -0,0 +1,18 @@
+# Changing host connectivity
```

Comment:
Should be address for node (the host is the object on the storage).

Reply:
@roysahar  I'm not sure what this is being referred to. please specify. thanks

Reply:
```suggestion
# Changing node connectivity
```

Reply:
@roysahar  is that what you were referring to (what ori suggested above)?

Reply:
Yes, thanks @oriyarde

Reply:
LGTM. will let @roysahar resolve when they see fit